resources/scss/bootstrap/_pagination.scss

```diff
@@ -1,9 +1,14 @@
 .pagination {
     display: inline-block;
     padding-left: 0;
-    margin-top: $spacer-y;
-    margin-bottom: $spacer-y;
+    list-style-type: none;
+    margin: $spacer-y auto $spacer-y auto;
     @include border-radius();
+
+    li {
+        display: inline;
+        padding: 5px 10px;
```

Comment:
Bah didn't see this before I merged. `em` (Ill do them though I suppose since Im on now ^_^)